### PR TITLE
MET Landing

### PR DESCRIPTION
**Pull request by lilia-obushenko**
-[DEMO LINK](https://lilia-obushenko.github.io/layout_miami/)

### CODE REVIEW

File: src/styles/blocks/menu.scss [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+    transition: border-color, transform 0.3s;
+
+    @include onDesktop {
+      &:hover {
+        border-color: $colorWhite;
+        transform: scale(120%);
+      }
+    }
+
```
Comment (Bogdan-Mykhailov) [inline src/styles/blocks/menu.scss:54]: empty line but it doesn't meter  coz this pr are closed

**Review by Bogdan-Mykhailov — COMMENTED** [review]
mate platform sent me to review this pool request but it already closed

**Review by valeriiayarzhemska — COMMENTED** [review]
Platform sent me here, bit its closed

**Review by valeriiayarzhemska — COMMENTED** [review]
.

**Pull request closed without merging**